metriken-exposition/Cargo.toml

```diff
@@ -22,6 +22,7 @@ serde = { version = "1.0.196", features = ["derive"], optional = true }
 serde_json = { version = "1.0.114", optional = true }
 
 [features]
+default = ["parquet"]
```

Comment:
Do we want the default to be parquet or parquet-conversion?

Reply:
neither! This is leftover from testing.